### PR TITLE
News fix

### PR DESCRIPTION
**Pull request by dieie32**

### CODE REVIEW

File: modules/news/news.service.js [hunk 1]
```diff
@@ -26,22 +27,63 @@ class NewsService {
...
     }
     return await News.findByIdAndUpdate(id, news, { new: true });
   }
 
-  async addNews(data) {
+  async addNews(data, upload) {
     if (await this.checkNewsExist(data)) {
       throw new Error(NEWS_ALREADY_EXIST);
     }
+    if (upload[0] && upload[1]) {
```
Comment (moran711) [inline modules/news/news.service.js:66]: You need to upload both photos here.

File: modules/news/news.service.js [hunk 2]
```diff
@@ -26,22 +27,63 @@ class NewsService {
...
   }
 
-  async updateNews(id, news) {
+  async updateNews(id, news, upload) {
     const foundNews = await News.findById(id);
     if (!foundNews) {
       throw new Error(NEWS_NOT_FOUND);
     }
+    if (upload[0] && upload[1]) {
+      const uploadResultAuthor = await uploadService.uploadFile(upload[0], [
```
Comment (moran711) [inline modules/news/news.service.js:36]: You can move this to utils file in news.

**Review by ivan-bonk — APPROVED** [review]

**Review by Iryna-Bzdyr — APPROVED** [review]

File: modules/news/news.service.js [hunk 3]
```diff
@@ -26,22 +28,35 @@ class NewsService {
...
   }
 
-  async updateNews(id, news) {
+  async updateNews(id, news, upload) {
     const foundNews = await News.findById(id);
     if (!foundNews) {
       throw new Error(NEWS_NOT_FOUND);
     }
 
+    if (upload[0] && upload[1]) {
```
Comment (moran711) [inline modules/news/news.service.js:37]: You also need to delete previous image.

File: modules/news/news.service.js [hunk 4]
```diff
@@ -26,22 +28,35 @@ class NewsService {
...
     return await News.findByIdAndUpdate(id, news, { new: true });
   }
 
-  async addNews(data) {
+  async addNews(data, upload) {
     if (await this.checkNewsExist(data)) {
       throw new Error(NEWS_ALREADY_EXIST);
     }
+
+    data.author.image = await uploadLargeImage(upload[0]);
```
Comment (moran711) [inline modules/news/news.service.js:57]: You can check if you have something in upload. If there are no files you can throw error.

**Pull request merged**